src/prefect/environments/storage/bytes.py

```diff
@@ -0,0 +1,24 @@
+import prefect
+from prefect.environments.storage import Storage
+
+
+class Bytes(Storage):
+    """
+    Bytes storage.
+    """
+
+    def __init__(self) -> None:
+        pass
+
+    def build(self, flow: "prefect.Flow") -> "Storage":
+        """
+        Build the Bytes storage object.
+
+        Args:
+            - flow (prefect.Flow): Flow to be stored
+
+        Returns:
+            - Bytes: a new Bytes storage object that contains information about how and
+                where the flow is stored.
+        """
+        return Bytes()
```

Comment:
```
# TODO: implement this
```
?

Reply:
This is also one of the issues I will be opening